SmartHashCat/shc_filter/filter_rule_format.py

```diff
@@ -0,0 +1,25 @@
+from shc_filter.filter_abstract import FilterAbstract
+
+
+class Filter(FilterAbstract):
+
+    def __init__(self, attacker, previous_input):
+        super(Filter, self).__init__(previous_input)
+
+    def transform_to_prepend_hashcat_format(self, word):
+        to_return = ""
+        for c in word[::-1]:
+            to_return += "^"+c
```

Comment:
For better performance, use list.append() with format-string:
 ```
        to_return = []
        for c in word[::-1]:
            to_return.append(f'^{c}')
        return ''.join(to_return)
```

Reply:
Done in #28